src/components/CreateNewEvidenceForm.tsx

```diff
@@ -60,19 +55,25 @@ const SearchForm = () => {
         setIsSubmitting(false);
         return;
       }
-        
-    response.json().then((data) => {
-      if (data.duplicate) {
-        window.location.href = `/card/${data.card_metadata.id}?collisions=${data.duplicate}`;
-        return;
-      }
-      window.location.href = `/card/${data.card_metadata.id}`;
-      return;
-    });
 
-      
+      void response.json().then((data) => {
+        if (!response.ok) {
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+          if (data.duplicate) {
```

Comment:
we will still get a 200 response if its a duplicate it just has the duplicate flag set